### ISSUE TITLE
openshift-apiserver drops out temporarily after install.

### ISSUE THREAD

**Issue by RobertKrawitz** [openshift/installer#813]
# Version

```/home/rkrawitz/go/src/github.com/openshift/installer/bin/openshift-install v0.5.0-master-36-gb4f5ceb6bfde8d3dc0e29f708e0494488ea37ee0
ostree 47.194
Installer b4f5ceb6bfde8d3dc0e29f708e0494488ea37ee0
Terraform v0.11.8
```

# Platform (aws|libvirt|openstack):

AWS and libvirt

# What happened?

After the installer completed (immediately on AWS, about 7 minutes later on libvirt), I was able to `oc login`.  A few minutes later, I was no longer able to; I got 400 messages on the command line.  Shortly thereafter (within a few minutes), I was then able to interact, and it stayed up.

On AWS, I was able to narrow the time down to within maybe 20 seconds prior to 20:09:01 UTC.

# What you expected to happen?

Once I'm able to log in, I expect to be able to interact with either the CLI or the console without interruption.

# How to reproduce it (as minimally and precisely as possible)?

1. After the installer completes, run `oc login -u kubeadmin -p <password>` until the login succeeds.

2. Run `oc get routes --all-namespaces` (or any similar command) repeatedly.

3. After several minutes, observe

```$ oc get routes --all-namespaces
No resources found.
error: You must be logged in to the server (Unauthorized)
```
If you try to run `oc login`, observe

```$ oc login -u kubeadmin -p "$OPENSHIFT_INSTALL_PASSWORD"
Error from server (InternalError): Internal error occurred: unexpected response: 400
```
4. Wait a few more minutes (no need to log in again), and you'll be able to run oc commands as normal, and it stays up.

**Comment by deads2k** [comment]
On libvirt an api server drop is expected.

On aws, I wouldn't.  Based on the message and the user, this worth checking with @enj for the new authenticator.

**Comment by RobertKrawitz** [comment]
This is very definitely happening on AWS also.

**Comment by wking** [comment]
I think this might have been resolved by openshift/machine-config-operator#225.  Can anyone still reproduce?  If not, can we close this?

**Comment by RobertKrawitz** [comment]
I didn't observe it on either aws or libvirt; go ahead and close it.
